tests/parser/TableManagerTests.cpp

```diff
@@ -1357,3 +1358,47 @@ BOOST_AUTO_TEST_CASE( TestParseTABDIMS ) {
     Opm::Parser parser;
     BOOST_CHECK_NO_THROW( parser.parseString(data, Opm::ParseContext()));
 }
+
+BOOST_AUTO_TEST_CASE (Regdims_Entries) {
+
+    // All defaulted
+    {
+        const auto input = std::string {
+            R"~(
```

Comment:
What is the `~(` syntax?

Reply:
> What is the `~(` syntax?

It's a "delimiter" as outlined in item 6 of [cppreference's "string literal" documentation](http://en.cppreference.com/w/cpp/language/string_literal).  It is mainly intended to disambiguate the case of the character sequence `)"` legitimately occurring within the string itself.  That would otherwise need an escape sequence.  I can remove it if you think it's distracting.

Reply:
> I can remove it if you think it's distracting.

No no - I just wondered.